PermissionsEx/permissions.yml

```diff
@@ -765,6 +765,56 @@ groups:
       suffix: '&4'
     inheritance:
     - mortal
+    worlds:
+      survival:
+        permissions:
+        - powers.fireballpunch
+        - powers.wallrun
```

Comment:
i cant help you, this was not made under my account, sorry :\

You should have done each hero separately. next time, do a PR for each hero, it makes it easier when errors happen.